app/assets/javascripts/dragdrop.js

```diff
@@ -33,25 +33,27 @@ jQuery(function() {
   });
   $('.dropzone').on('drop',function(e) {
     // this 'drop' listener is also reused for pages with just one form, ie. /wiki/new
-    $D.selected = $(e.target).closest('div.comment-form-wrapper').eq(0);
+    const closestCommentFormWrapper = e.target.closest('div.comment-form-wrapper'); // this returns null if there is no match
     e.preventDefault();
     let params = {};
```

Comment:
I rewrote this part for greater readability, using JavaScript's `closest` (as opposed to jQuery's method with the same name).

JavaScript's `closest` returns `null`, whereas jQuery's returns an object. I thought this would be more intuitive.